glean/src/cli.ts

```diff
@@ -236,6 +236,11 @@ function stopSpinner(spinner: NodeJS.Timeout) {
  * @param args the arguments passed to this process.
  */
 async function run(args: string[]) {
+  if (args.includes("--glean-parser-version")) {
+    console.log(GLEAN_PARSER_VERSION);
+    process.exit(1);
```

Comment:
Error exit? Should probably be 0.

```suggestion
    process.exit(0);
```

Reply:
Oops.